_includes/comparison.html

```diff
@@ -360,6 +354,31 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>yes</td>
       </tr>
     </tbody>
+    <thead style="background-color: white; color: #ff4d67">
+      <tr>
+        <th scope="col">Advantages & Drawbacks</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Advantages</th>
+        <td>Istio can be adapted and extended like no other Mesh. Thanks to it's big and vivid community, it's constantly improved. Istio is <a href="https://istio.io/about/community/customers/" target="_blank">used in production.</a></td>
+        <td>Linkerd 2 is designed to be non-invasive and is optimized for performance and usability. It therefore requires little time to adopt. Linkerd 2 is <a href="https://github.com/linkerd/linkerd2/blob/master/ADOPTERS.md" target="_blank">used in production.</a></td>
+        <td>AWS App Mesh is integrated in the AWS landscape and it is fully managed for you.</td>
+        <td>Consul Connect is well integrated with Consul and the Proxy can be changed and extended.</td>
```

Comment:
As it uses Consul it should be possible to run it on any Docker scheduler, right? Is that worth mentioning?

Reply:
Yep, good point. I'll add a sentence there too